.circleci/config.yml

```diff
@@ -342,6 +342,8 @@ jobs:
             # persistent workspace and to avoid any confusion with the version
             # we are trying to test.
             rm -Rf `find -name "emscripten"`
+            echo "V8_ENGINE = '`pwd`/upstream/bin/d8'" >> ~/.emscripten
+            echo "JS_ENGINES = [NODE_JS, V8_ENGINE]" >> ~/.emscripten
```

Comment:
Can this be reverted now?

Reply:
I think it's still useful for two things: we do test it on some `other` tests (which iterate on JS_ENGINES) which makes sure we remain compatible with the `d8` shell. Also I just realized while talking to @tlively that we *do* have simd tests that use d8 if present, and this pr did run them and they passed :)

Reply:
Once SIMD in node stabilizes I should probably update the simd tests to run in node as well.